github/repos.go

```diff
@@ -620,6 +620,23 @@ func (s *RepositoriesService) GetBranchProtection(owner, repo, branch string) (*
 	return p, resp, err
 }
 
+//GetProtectionRequest provides the ProtectionRequest struct to the caller
+func (s *RepositoriesService) GetProtectionRequest() *ProtectionRequest {
+	//TODO: Default values to be set, not thought about it yet
+	input := &ProtectionRequest{
+		RequiredStatusChecks: &RequiredStatusChecks{
+			IncludeAdmins: Bool(true),
+			Strict:        Bool(true),
+			Contexts:      &[]string{"continuous-integration"},
+		},
+		Restrictions: &BranchRestrictionsRequest{
+			Users: &[]string{"u"},
+			Teams: &[]string{"t"},
+		},
+	}
+	return input
+}
```

Comment:
> I have made these changes and also committed them. Are you able to see these changes. Am i supposed to create a new PR?

You shouldn't need to make a new PR.

After you've committed the changes, make sure to push them to the remote `master` master branch of your fork https://github.com/varadarajana/go-github.

See https://help.github.com/articles/pushing-to-a-remote/ for additional information.